projects/services-ogc/assets/testGeoJson.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
What should we do with test data?
We have multiple of this in different projects, maybe we should create a project for them and import them as dependency?

Reply:
yes!

Reply:
I only created a shared folder not an npm package because i think we should not publish this on npm.

78bddd9b932408f8ebaca6812d10044b2865e0c0